EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -6,6 +6,20 @@
 	objectVersion = 46;
 	objects = {
 
+/* Begin PBXAggregateTarget section */
+		C88F9A2C1F39A5E000704BFF /* Release */ = {
+			isa = PBXAggregateTarget;
+			buildConfigurationList = C88F9A2F1F39A5E000704BFF /* Build configuration list for PBXAggregateTarget "Release" */;
+			buildPhases = (
+				C88F9A321F39A5F100704BFF /* ShellScript */,
+			);
+			dependencies = (
+			);
+			name = Release;
```

Comment:
conceptually since we had couple different ways static / dynamic, that's why I suggested naming to be explicit. As this is not creating the full Cocoapods zip then it's fine the way it is.